src/components/Textarea/Textarea.tsx

```diff
@@ -99,6 +105,28 @@ const Textarea = ({
   const defaultTextAreaId = useId();
   const textAreaId = id || defaultTextAreaId;
 
+  const handleKeyDown = useCallback(
+    (event: KeyboardEvent) => {
+      const textareaElement = document.getElementById(textAreaId);
+      if (
+        event.key === "Enter" &&
+        (event.ctrlKey || event.metaKey) &&
+        document.activeElement === textareaElement
```

Comment:
This line (114) allows for the CTRL+Enter function to only occur when the TextArea is in focus.